plasmapy/__init__.py

```diff
@@ -77,22 +78,8 @@ def online_help(query):
 
     webbrowser.open(url)
 
-__citation__ = """@misc{plasmapy_community_2018_1238132,
-  author       = {{PlasmaPy Community} and
-                  Murphy, Nicholas A. and
-                  Leonard, Andrew J. and
-                  Sta\'nczak, Dominik and
-                  Kozlowski, Pawel M. and
-                  Langendorf, Samuel J. and
-                  Haggerty, Colby C. and
-                  Beckers, Jasper P. and
-                  Mumford, Stuart J. and
-                  Parashar, Tulasi N. and
-                  Huang, Yi-Min},
-  title        = {{PlasmaPy: an open source community-developed 
-                   Python package for plasma physics}},
-  month        = apr,
-  year         = 2018,
-  doi          = {10.5281/zenodo.1238132},
-  url          = {https://doi.org/10.5281/zenodo.1238132}
-}"""
+
+__citation__ = [
+    "https://doi.org/10.5281/zenodo.1238132",
+    "https://doi.org/10.5281/zenodo.3235817",
+]
```

Comment:
I'm of two minds about this one - on the one hand, this is the first usage of `__citation__` I've seen that isn't basically a BibTeX entry. On the other, this provides easy (if multi-step) access to other citation methods here.

I guess it's fine, but I'll just note how much I'd love it to be able to do something like the following

```python
for imported_package in ag("~/scientific_software", "import (.*)"):
   try:
       print(imported_package.__citation__)
   except AttributeError:
       pass
```

for future publications. But that's way out of scope of this issue :smiley_cat:

Reply:
I was looking this up recently if there were any standards and came across https://github.com/adrn/CitationPEP/issues/2 which suggests creating a type for citations that can handle the different citation possibilities.  The loop you suggested could also probably be done if CodeMeta files become standardized and widely used (#622).